gluten-iceberg/src/test/scala/io/glutenproject/execution/VeloxIcebergSuite.scala

```diff
@@ -56,44 +71,246 @@ class VeloxIcebergSuite extends WholeStageTransformerSuite {
     }
   }
 
-  test("iceberg partitioned table") {
-    withTable("p_str_tb", "p_int_tb") {
+  test("iceberg bucketed join") {
+    assume(isSparkVersionAtleast("3.4"))
```

Comment:
So these tests doesn't work for Spark3.3? should we fix them, maybe in late PR?

Reply:
Yea, in my view the storage partitioned join was usable mostly from Spark 3.4. So, I didn't bother making it work for 3.3. Let me know if you think some users are using spj on 3.3 and I can look more. Maybe in 3.5 follow up I will add some documentation on README about SPJ support.

Reply:
Let's add the limitation to Readme that it only work on Spark3.2 now. The other gluten user may add support to 3.3 or later if they need